src/rpc/masternode.cpp

```diff
@@ -145,7 +145,7 @@ UniValue masternode(const JSONRPCRequest& request)
                 "  check        - Force check all masternodes and remove invalid ones\n"
                 "  count        - Get information about number of masternodes (DEPRECATED options: 'total', 'ps', 'enabled', 'qualify', 'all')\n"
                 "  current      - Print info on current masternode winner to be paid the next block (calculated locally)\n"
-                "  genkey       - Generate new masternodeprivkey\n"
+                "  genkey       - Generate new masternodeprivkey (optional param: boolean, optional, default=false) generate compressed privkey\n"
```

Comment:
maybe `, optional param: 'compressed' - (boolean, optional, default=false) generate compressed privkey` ?

Reply:
That works.